feat: support multiple templates in placeholders (#193)
AmadeusITGroup/otter: docs/rules-engine/how-to-use/placeholders.md

* Html limited to angular sanitizer supported behavior
* Urls (relative ones will be processed to add the dynamic-media-path)
* Facts references

### Static localization
The first choice you have when you want to localize your template is the static localization.
You need to create a localized template for each locale and provide the template Url with [LANGUAGE] (ex: assets/placeholders/[LANGUAGE]/myPlaceholder.json)

> The rules engine service will handle the replacement of [LANGUAGE] for you, and when you change lang a new call will be performed to the new 'translated' Url.

Reviewer: ```suggestion
The rules engine service will handle the replacement of [LANGUAGE] for you, and when you change language a new call will be performed to the new 'translated' URL.
```

> Note that the URL caching mechanism is based on the url NOT 'translated', meaning that if you change from en-GB to fr-FR then you decide to switch back all the calls will be done again.

Reviewer: ```suggestion
Note that the URL caching mechanism is based on the url NOT 'translated', meaning that if you change from en-GB to fr-FR then you decide to switch back and all the calls will be made again.
```

This behavior is based on the fact that a real user rarely go back and force with the language update.

### Dynamic localization

### Multiple templates in same placeholder

> You can use placeholder actions to target the same placeholderId with different templates URLs.

Reviewer: ```suggestion
You can use placeholder actions to target the same placeholderId with different template URLs.
```
